### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by D0LeD**

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package main
+
+func TestIsValidNumber(t *testing.T){
+	for _, tc := range []struct {
+		testnum int
```
Comment (zayac) [inline main_test.go:7]: optional: `testnum` are meaningless, because they do not provide any test description. I'd provide a meaningful `string` description instead.

File: main_test.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package main
+
+func TestIsValidNumber(t *testing.T){
```
Comment (zayac) [inline main_test.go]: Did you run the tests? They don't compile.

File: main_test.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import "testing"
+
+func TestIsValidNumber(t *testing.T) {
+	for _, tc := range []struct {
+		testnum int
+		num     string
+		base    int
+		want    bool
+	}{
+		{testnum: 1, num: "", base: 0, want: true},
```
Comment (zayac) [inline main_test.go:12]: Can you add a test for num = "" and base >= 2? You program must output false, because an empty string is an invalid number.
